server/proxy/modules/capture/cap_main.c

```diff
@@ -127,7 +127,7 @@ static BOOL capture_plugin_session_end(proxyData* pdata)
 	wStream* s;
 
 	socket = capture_plugin_get_socket(pdata);
-	if (socket == -1)
+	if ((INT64)socket == -1LL)
```

Comment:
I think using `if (socket == INVALID_SOCKET)` would be more clear and avoids the casting